CalculatorTests/CalculatorTests/Tests/CalculatorPageTests.cs

```diff
@@ -18,9 +18,34 @@ public void SetUp()
         {
             var options = new ChromeOptions { AcceptInsecureCertificates = true };
             driver = new ChromeDriver(options);
+            SetSettingsToDefault();
             driver.Url = "https://localhost:5001/Calculator";
         }
 
+
+        public void SetSettingsToDefault()
+        {
+            driver.Url = "https://localhost:5001/Settings";
+
+            IWebElement dateFormatField = driver.FindElement(By.Id("dateFormat"));
+            IWebElement numberFormatField = driver.FindElement(By.Id("numberFormat"));
+            IWebElement defaultCurrencyField = driver.FindElement(By.Id("currency"));
+            IWebElement saveButton = driver.FindElement(By.Id("save"));
+            SelectElement dateFormatDropdown = new SelectElement(dateFormatField);
+            SelectElement numberFormatDropdown = new SelectElement(numberFormatField);
+            SelectElement defaultCurrencyDropdown = new SelectElement(defaultCurrencyField);
+
+            new WebDriverWait(driver, TimeSpan.FromSeconds(200))
+             .Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementToBeClickable(By.Id("save")));
+            dateFormatDropdown.SelectByText("dd/MM/yyyy");
+            numberFormatDropdown.SelectByText("123,456,789.00");
+            defaultCurrencyDropdown.SelectByText("$ - US dollar");
+            saveButton.Click();
+            System.Threading.Thread.Sleep(2000);
```

Comment:
Try to replace the 'Thread.Sleep' with Wait. It should be such condition, as 'wait for alert is shown'

Reply:
Fixed